RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -60,7 +62,11 @@ struct PurchaseInformation {
             self.expirationOrRenewal = entitlement.expirationOrRenewal(dateFormatter: dateFormatter)
             self.productIdentifier = entitlement.productIdentifier
             self.store = entitlement.store
-            self.price = entitlement.priceBestEffort(product: subscribedProduct)
+            if let renewalPrice {
+                self.price = renewalPrice
+            } else {
+                self.price = entitlement.priceBestEffort(product: subscribedProduct)
```

Comment:
Fall back to our old calculation method if the RenewalInfo method failed for some reason